app/components/npq_separation/navigation_structures/guidance_navigation_structure.rb

```diff
@@ -27,6 +27,11 @@ def structure
             href: api_guidance_page_path(page: "/", anchor: "how-to-guides"),
             prefix: "/api/guidance/",
           ) => [],
+          Node.new(
+            name: "Process diagrams",
+            href: api_guidance_page_path(page: "/", anchor: "process-diagrams"),
+            prefix: "/api/guidance/",
```

Comment:
Think prefix needs to be different from how-to guides so it can be highlighted when the page is browsed.